tests/unit/test_nuts_generation.py

```diff
@@ -0,0 +1,7 @@
+from ted_data_eu.services.etl_pipelines.ted_data_etl_pipeline import generate_nuts_level
+
+
+def test_nuts_generation(real_nuts, fake_nuts):
```

Comment:
Surely there are more cases of NUTS. 
Can you try Copilot to write the assertions for you if you give it a few examples of NUTS codes?